app/frontend/js/modals/updateExtractionDefinitionModal.js

```diff
@@ -0,0 +1,63 @@
+import { each } from "lodash";
+
+const updateExtractionDefinitionModal = document.getElementById(
+  "update-extraction-definition-modal"
+);
+
+if (updateExtractionDefinitionModal) {
+  const extractionDefinitionFormat = document.getElementById(
+    "js-extraction-definition-format"
+  );
+  const splitDropdown = document.getElementById(
+    "js-extraction-definition-split-dropdown"
+  );
+
+  if (splitDropdown != null) {
+    toggleSplitElements(extractionDefinitionFormat.value);
+
+    extractionDefinitionFormat.addEventListener("change", (event) => {
+      toggleSplitElements(event.target.value);
+    });
+
+    splitDropdown.addEventListener("change", () => {
+      toggleSplitElements(extractionDefinitionFormat.value);
+    });
+  }
+
+  function toggleSplitElements(format) {
```

Comment:
Might be good to add a comment saying what this is doing. It isn't immediately clear to me what this means.